spec/requests/in_progress_forms_request_spec.rb

```diff
@@ -362,6 +362,32 @@ def response_body
 
           expect(existing_form.reload.form_data).to eq(form_data)
         end
+
+        context 'has checked \'One or more of my rated conditions that have gotten worse\'' do
+          let(:form_data) { { 'view:claim_type': { 'view:claiming_increase': true } }.to_json }
+
+          before { allow(StatsD).to receive(:increment) }
+
+          it 'updates form and includes cfiMetric in metadata, and logs metric' do
+            put v0_in_progress_form_url(existing_form.form_id),
+                params: { form_data:, metadata: existing_form.metadata }
+            expect(response).to have_http_status(:ok)
+            expect(existing_form.reload.metadata.keys).to include('cfiMetric')
+            expect(StatsD).to have_received(:increment).with('ee.max-cfi.on.rated-disabilities')
+          end
+        end
+
+        context 'has not checked \'One or more of my rated conditions that have gotten worse\'' do
+          before { allow(StatsD).to receive(:increment) }
+
+          it 'updates form and does not include cfiMetric in metadata' do
+            put v0_in_progress_form_url(existing_form.form_id),
+                params: { form_data:, metadata: existing_form.metadata }
+            expect(response).to have_http_status(:ok)
+            expect(existing_form.reload.metadata.keys).not_to include('cfiMetric')
+            expect(StatsD).not_to have_received(:increment).with('ee.max-cfi.on.rated-disabilities')
+          end
+        end
```

Comment:
This is such a cool test!

Since there's a bit of flux in exactly what we're logging, I'll do a closer think about the testing coverage and strategy in the next pass.

Reply:
Thanks!